### DISCUSSION TITLE
copying of fan profiles + inconsistency question

### DISCUSSION THREAD

**Discussion by patrickstar13** [seerge/g-helper#2883] [General]
hi i tried copying profiles to play around with them a little but couldnt manage to do it in the config. my thought was to copy this block

`"fan_profile_gpu_6": "32-41-46-4B-50-55-5A-5F-02-10-12-18-1C-22-2A-3B",
  "fan_profile_cpu_6": "32-41-46-4B-50-55-5A-5F-02-0F-13-18-1B-1F-28-3B",
  "limit_cpu_6": 23,
  "limit_fast_6": 35,
  "limit_total_6": 35,
  "limit_slow_6": 35,
  "mode_name_6": "steady",
  "mode_base_6": 2,
  "auto_apply_power_6": 1,
  "auto_apply_6": 1,
  "gpu_clock_limit_6": 3000,
  "powermode_6": "00000000-0000-0000-0000-000000000000",
  "auto_uv_6": 0,
  "cpu_temp_6": 98,
  "cpu_uv_6": 0,
  "igpu_uv_6": 0`

and replace the 6 with 7 and enter a new string for the name but it doesnt work like that somehow.
i know i could just copy and set them in ghelper but i am a lazy man :D

as for my further question, i noticed, like in the block above, some limits set in ghelper dont seem to match the entrys in the config, 
`  "limit_cpu_6": 23,`  although set to 35 in ghelper

also, from another profiles, 
`  "limit_total_4": 35,
  "limit_slow_4": 35,
  "limit_cpu_4": 30,
  "limit_fast_4": 35,`

`  "limit_total_3": 35,
  "limit_slow_3": 35,
  "limit_cpu_3": 28,
  "limit_fast_3": 35,`

although again everything set to 35. should i just correct it in the config to my set values in ghelper? (35)

**Comment by seerge** [comment]
@patrickstar13 what are you trying to achieve ?

`limit_cpu_6` is relevant only for all AMD models (that have platform + CPU only limits) for Nvidia devices it's ignored

`limit_total` is SPL, `limit_slow` is SPPT, `limit_fast` is FPPT

**Comment by patrickstar13** [comment]
omg its the god himself, thx for making our laptops so much cooler in this hot summer!

on step 1 i tried to just copy pasty some fan profiles within the profile.cfg so i dont have to redraw the lines for a similar fan profile on top of the old one.
i tried that by copypasting the above lines and relaunching ghelper but i couldnt duplicate one profile with another name(it just wasnt recognized in ghelper as a new fan profile)

on step 2 ok thx solved, so its irrelevant. my first guess was that it was a value for the SPL sPPt fPPPT settings which i saw was not at 35 like the rest but something around that number and that made me curious. i do have a g15 (ryzen + nvidia gpu)

**Reply by seerge** [reply]
If you by some reason want to clone profiles, I would first create them at least somehow using (+) button, set power limits manually and then (if it's so difficult to re-create a fan curve) just quit app, edit config by copying ONLY `fan_profile_` values to the newly created profile and start app again :) 

As everything except fan curves are just simple sliders that you can set in a few seconds :)

**Reply by patrickstar13** [reply]
nice just did that, worked like a charm

now i can make my perfect profiles for each task

thx again for everything. i am chilling at 60° instead of 82° cuz of ghelper, its actually insane O_o
